libs/dal/src/repositories/message/message.entity.ts

```diff
@@ -54,4 +54,6 @@ export class MessageEntity {
 
   @Exclude()
   providerResponse: any; // eslint-disable-line @typescript-eslint/no-explicit-any
+
+  triggerPayload: any; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
```suggestion
  triggerPayload: Record<string, unkown>; 
```

@djabarovgeorge  Do we also have a @novu/shared interface for this used by the notification-center component?

Reply:
Hey @scopsy, Could you elaborate more on what you mean? I am not sure I understood correctly.

In addition, what do you think regarding the naming of `triggerPayload` if it sounds correct, or is it obvious that its triggers payload and we should rename it to `payload`

Reply:
Good questions regarding the naming, I think I would go with payload. Since in the future we will allow the user to modify it using the API (needed when we will implement buttons and actions).

Regarding the interfaces, I've seen you only added them in the DAL. So this object won't be available in the interface the client-side uses which comes from @novu/shared. I think you need to add it there aswell.

And about `any` type, I would just try to use something else rather than `any`. :)

Reply:
Noted let me update it pr :)